prototype_source/lite_interpreter.rst

```diff
@@ -28,53 +32,33 @@ Get ImageSegmentation demo app in Android: https://github.com/pytorch/android-de
     # Export lite interpreter version model (compatible with lite interpreter)
     scripted_module._save_for_lite_interpreter("deeplabv3_scripted.ptl")
```

Comment:
The `optimize_for_mobile` call is strongly recommended:
```
from torch.utils.mobile_optimizer import optimize_for_mobile
optimized_scripted_module = optimize_for_mobile(scripted_module)
optimized_scripted_module._save_for_lite_interpreter("deeplabv3_scripted_optimized.ptl")
```

In my test, using `deeplabv3_scripted_optimized.ptl`, the inference time speeds up about 100%, from over 3s (for deeplabv3_scripted.ptl` to about 1.5s.

Reply:
Oh good catch. Will update it.

Reply:
Update in the last commit